cassdegrees/templates/createsubplan.html

```diff
@@ -6,6 +6,54 @@
 
 {% block content %}
 
-<p>This is a test</p>
+    {% if msg != None %}
+        <p class="{% if is_error %}msg-error{% else %}msg-success{% endif %}">{{ msg }}</p>
+    {% endif %}
+
+    <form class="anuform" action="/create_subplan/" method="post">
+
+        {% csrf_token %}
+
+        <fieldset>
+
+            <legend>Subplan details</legend>
+
+            <!-- As HTML forms only support get or post methods,
+            add this hidden input for consistency with the rest of the forms -->
+            <input type="hidden" name="_method" value="post">
+
+            <p>
+                <label for="code">Subplan Code:</label>
+                <input class="text" id="code" type="text" name="code"
+                       aria-required="true" required value="{% if code != None %}{{code}}{% endif %}">
```

Comment:
```suggestion
                       aria-required="true" required value="{{ code|default:"" }}">
```

Simplified version based on  Django standard